### PR TITLE
Enable automation test for all feature store notebooks

### PR DESCRIPTION
**Pull request by fredms**
# Description

# Checklist

- [x] I have read the [contribution guidelines](https://github.com/Azure/azureml-examples/blob/main/CONTRIBUTING.md)
- [ ] I have coordinated with the docs team (mldocs@microsoft.com) if this PR deletes files or changes any file names or file extensions.
- [x] Pull request includes test coverage for the included changes.
- [ ] This notebook or file is added to the [CODEOWNERS](https://github.com/Azure/azureml-examples/blob/main/.github/CODEOWNERS) file, pointing to the author or the author's team.

### CODE REVIEW

File: sdk/python/featurestore_sample/notebooks/sdk_only/1. Develop a feature set and register with managed feature store.ipynb [hunk 1]
```diff
@@ -287,7 +287,8 @@
     "featurestore_name = \"my-featurestore\"\n",
     "featurestore_location = \"eastus\"\n",
     "featurestore_subscription_id = os.environ[\"AZUREML_ARM_SUBSCRIPTION\"]\n",
-    "featurestore_resource_group_name = os.environ[\"AZUREML_ARM_RESOURCEGROUP\"]"
+    "featurestore_resource_group_name = os.environ[\"AZUREML_ARM_RESOURCEGROUP\"]\n",
+    "feature_version = \"<FEATURE_VERSION>\""
```
Comment (qjxu) [inline sdk/python/featurestore_sample/notebooks/sdk_only/1. Develop a feature set and register with managed feature store.ipynb:291]: what is feature_version?

Comment (qjxu) [reply sdk/python/featurestore_sample/notebooks/sdk_only/1. Develop a feature set and register with managed feature store.ipynb:291]: Is it not visible to user?

**Review by qjxu — APPROVED** [review]

File: sdk/python/featurestore_sample/notebooks/sdk_only/1. Develop a feature set and register with managed feature store.ipynb (hunk 1 above)
Comment (fredms) [reply sdk/python/featurestore_sample/notebooks/sdk_only/1. Develop a feature set and register with managed feature store.ipynb:291]: all values with "<>" are supposed to be replaced with user defined values when user execute the notebooks.

Comment (qjxu) [reply sdk/python/featurestore_sample/notebooks/sdk_only/1. Develop a feature set and register with managed feature store.ipynb:291]: Can you rename the constant FEATURE_VERSION to VERSION? As it is used for featureset and also for featurestore entity.

File: sdk/python/featurestore_sample/setup-resources.sh [hunk 2]
```diff
@@ -8,18 +8,23 @@ AML_WORKSPACE_NAME=$(az configure -l --query "[?name=='workspace'].value" -o tsv
 OUTPUT_COMMAND="print"
 FEATURE_STORAGE_ACCOUNT_NAME=${RESOURCE_GROUP}fs
```
Comment (qjxu) [inline sdk/python/featurestore_sample/setup-resources.sh:9]: For my education, which version of control plane and data plane sdk are used for test? where it is defined?

File: sdk/python/featurestore_sample/notebooks/sdk_only/1. Develop a feature set and register with managed feature store.ipynb (hunk 1 above)
Comment (fredms) [reply sdk/python/featurestore_sample/notebooks/sdk_only/1. Develop a feature set and register with managed feature store.ipynb:291]: make sense. I'll make the change

**Review by bastrik — APPROVED** [review]

**Pull request merged**